index.html

```diff
@@ -2974,39 +3013,36 @@ <h3>Receiving and parsing content</h3>
             the <var>watch</var> was registered on.
           </li>
           <li>
-            Let <var>options</var> be the <code><a>NFCWatchOptions</a></code>
-            saved with <var>watch</var>.
-          </li>
-          <li>
-            If <var>options</var>.mode is <code>"web-nfc-only"</code> and
+            If <var>reader_instance</var>.[[\Accept]] is <code>"web-nfc-only"</code> and
             <var>message</var>.url is <var>null</var>, skip to the next
             <a>NFC watch</a>.
           </li>
           <li>
-            Otherwise, if <var>options</var>.mode is <code>"any"</code>,
-            set <var>options</var>.url to <code>""</code>.
+            Otherwise, if <var>reader_instance</var>.[[\Accept]] is <code>"any"</code>,
+            set <var>reader_instance</var>.[[\Url]] to <code>""</code>.
           </li>
           <li>
-            If the <a>URL pattern</a> <var>options</var>.url is not
+            If the <a>URL pattern</a> <var>reader_instance</var>.[[\Url]] is not
             an empty <code>String</code> invoke
             <a href="#dfn-match-web-nfc-id-with-url-pattern">URL pattern match</a>
-            algorithm, providing <var>options</var>.url as a <a>URL pattern</a> and
+            algorithm, providing <var>reader_instance</var>.[[\Url]] as a
+            <a>URL pattern</a> and
             <var>message</var>.url as a <code><a>Web NFC Id</a></code>. If
             <a href="#dfn-match-web-nfc-id-with-url-pattern">URL pattern match</a>
             algorithm returns <code>false</code>, skip to the next <a>NFC watch</a>.
           </li>
           <li>
-            If <var>options</var>.recordType is <a href=#dfn-not-present>present</a> and
-            it is not equal to any <var>record</var>.recordType where <var>record</var>
+            If <var>reader_instance</var>.[[\RecordType] is <a href=#dfn-not-present>present</a> and
+            it is not equal to any <var>record</var>.[[\recordType]] where <var>record</var>
```

Comment:
Where is the type of these records defined?

Probably either both this line and line 3041 should use internal slot notation or property notation, but not one of each.

Reply:
These are internal slots on the NFCReader. But the NFCRecord (the data we read) has these as properties

Reply:
http://localhost:8000/#dom-nfcrecord

Reply:
Maybe we need to define the internal slots for the NFCReader somewhere. Can be a follow-up